pkg/util/version/version.go

```diff
@@ -58,3 +68,26 @@ func (v *Version) Lt(w *Version) bool {
 
 	return false
 }
+
+func (v *Version) Eq(w *Version) bool {
+	return bytes.Compare(v.V[:], w.V[:]) == 0
+}
+
+func GetUpgrade(v *Version) (*Upgrade, error) {
+	for _, u := range Upgrades {
+		if u.Version.V[0] == v.V[0] &&
+			u.Version.V[1] == v.V[1] {
+			return &u, nil
+		}
+	}
+	return nil, fmt.Errorf("upgrade for %d not found", v.V[:])
+}
+
+func GetLatest() (*Upgrade, error) {
+	for _, u := range Upgrades {
+		if u.Latest {
+			return &u, nil
+		}
+	}
+	return nil, fmt.Errorf("latest version not not found")
```

Comment:
We should always have only 2 upgrades. So this would become very ambiguous. I was not able to think about how to make it easy to "split" and make it readable and understandable. 

Added test to `TestLatest`.

We could have one more test to check if we have only single x.Y version path configured. But I think this all code base should go away soon when we decide what we want to do with our version management,

Reply:
+1 on what @m1kola's thinking.